selfdrive/can/process_dbc.py

```diff
@@ -39,48 +38,78 @@ def main():
     if dbc_mtime < out_mtime and template_mtime < out_mtime and this_file_mtime < out_mtime:
       continue #skip output is newer than template and dbc
 
-    msgs = [(address, msg_name, msg_size, sorted(msg_sigs, key=lambda s: s.name not in (b"COUNTER", b"CHECKSUM"))) # process counter and checksums first
+    msgs = [(address, msg_name, msg_size, sorted(msg_sigs, key=lambda s: s.name not in ("COUNTER", "CHECKSUM"))) # process counter and checksums first
             for address, ((msg_name, msg_size), msg_sigs) in sorted(can_dbc.msgs.items()) if msg_sigs]
 
     def_vals = {a: set(b) for a,b in can_dbc.def_vals.items()} #remove duplicates
     def_vals = [(address, sig) for address, sig in sorted(def_vals.items())]
 
     if can_dbc.name.startswith("honda") or can_dbc.name.startswith("acura"):
-      checksum_type = "honda"
+      car_type = "honda"
+      car_is_little_endian = False
       checksum_size = 4
+      counter_size = 2
     elif can_dbc.name.startswith("toyota") or can_dbc.name.startswith("lexus"):
-      checksum_type = "toyota"
+      car_type = "toyota"
+      car_is_little_endian = False
       checksum_size = 8
+      counter_size = None
+    elif can_dbc.name.startswith("vw") or can_dbc.name.startswith("volkswagen") or can_dbc.name.startswith("audi") or can_dbc.name.startswith ("seat") or can_dbc.name.startswith("skoda"):
+      car_type = "volkswagen"
+      car_is_little_endian = True
+      checksum_size = 8
+      counter_size = 4
     else:
-      checksum_type = None
+      car_type = None
+      car_is_little_endian = None
+      checksum_size = None
+      counter_size = None
 
     for address, msg_name, msg_size, sigs in msgs:
+      dbc_msg_name = dbc_name + " " + msg_name
       for sig in sigs:
-        if checksum_type is not None and sig.name == b"CHECKSUM":
```

Comment:
As part of the Python3 changes, these strings got converted to byte literals. I'm not sure why. I can tell you the comparisons were failing, so the code here at 59 (and 66, 72, 74, and probably 42) was never matching, so these checks were NOT being done at all in OP 0.6.5-devel.